Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -1647,6 +1649,13 @@ public IEnumerable<BaseData> GetNextTicks()
 
             foreach (var tick in ticks)
             {
+                // if we have a cached data time zone for this symbol, perform the conversion 
+                DateTimeZone dataTimeZone;
+                if (_dataTimeZones.TryGetValue(tick.Symbol.Value, out dataTimeZone))
+                {
+                    tick.Time = tick.Time.ConvertTo(TimeZones.NewYork, dataTimeZone);
```

Comment:
Meant data timezone; I'm assuming we convert it from data timezone to UTC later in the stack and that is why the data isn't arriving as expected (maybe the conversion takes it out of exchange hours & it gets ignored etc).

Regardless -- If the raw live ticks arrived in the same timezone as the raw backtesting files it would keep things consistent?